README.md

```diff
@@ -35,6 +35,22 @@ dependencies:
 
 Then run the command `flutter packages get` on the console.
 
+## Additional reactive fields
```

Comment:
While *fields* is right, I prefer *field widgets*:
Additional reactive field widgets